cmd/burrow/commands/start.go

```diff
@@ -36,16 +33,34 @@ func Start(output Output) func(cmd *cli.Cmd) {
 
 			output.Logf("Using validator address: %s", *conf.ValidatorAddress)
 
-			ctx, cancel := context.WithCancel(context.Background())
-			defer cancel()
-
-			kern, err := conf.Kernel(ctx, *restoreDumpOpt)
+			kern, err := core.NewKernel()
 			if err != nil {
 				output.Fatalf("could not create Burrow kernel: %v", err)
 			}
 
-			err = kern.Boot()
-			if err != nil {
+			if err = kern.ExecutorFromConfig(conf.Execution); err != nil {
+				output.Fatalf("could not add execution options: %v", err)
+			}
+
+			if err = kern.LoggerFromConfig(conf.Logging); err != nil {
+				output.Fatalf("could not configure logger: %v", err)
+			}
+
+			if err = kern.KeysFromConfig(conf.Keys); err != nil {
+				output.Fatalf("could not configure keys: %v", err)
+			}
+
+			tmConf := conf.Tendermint.TendermintConfig()
+			if err = kern.LoadState(conf.GenesisDoc, tmConf.DBDir()); err != nil {
+				output.Fatalf("could not load state: %v", err)
+			}
+
+			if err = kern.TendermintFromConfig(conf.Tendermint, nil, conf.ValidatorAddress); err != nil {
+				output.Fatalf("could not configure Tendermint: %v", err)
+			}
+
+			kern.AddProcesses(core.DefaultServices(kern, conf.RPC, conf.Keys)...)
+			if err = kern.Boot(); err != nil {
```

Comment:
I do see the logic of just having this wiring up logic in the CLI... However I feel like it might still be good to have a canonical location that shows you how to wire up a kernel.

My suggestion would be that we have a `core/config.go` that has all the `LoadXFromConfig` methods and we add a `LoadKernelFromConfig` which does the lot.